lib/app/service/docker.go

```diff
@@ -69,6 +69,17 @@ func exportLayers(
 	return nil
 }
 
+func excludeImagesStartingWith(images []string, s string) []string {
```

Comment:
```suggestion
func excludeImagesStartingWith(images []string, prefix string) []string {
```
Also, check for empty prefix at the start:
```go
func excludeImagesStartingWith(images []string, prefix string) []string {
	if prefix == "" {
		return images
	}
	i := 0 // i is a more popular index name :)
	for _, image := range images {
		if !strings.HasPrefix(image, prefix) {
			images[i] = image
			i++
		}
	}
	return images[:i]
}
```